build-octotiger.sh

```diff
@@ -34,8 +33,8 @@ ${CMAKE_COMMAND} \
     -DOCTOTIGER_WITH_CUDA=$OCT_WITH_CUDA \
     -DOCTOTIGER_WITH_KOKKOS=$OCT_WITH_KOKKOS \
     -DOCTOTIGER_WITH_BLAST_TEST=ON \
-    -DOCTOTIGER_WITH_TESTS=ON \
-    -DOCTOTIGER_WITH_Vc=ON \
+    -DOCTOTIGER_WITH_TESTS=OFF \
+    -DOCTOTIGER_WITH_Vc=OFF \
```

Comment:
I would keep these on ON as we don't gain anything by turning it OFF yet (as some old methods still pull in Vc)